store/src/main/java/Milk.java

```diff
@@ -0,0 +1,4 @@
+
+public class Milk extends Category {
+
```

Comment:
let's just create an object of Category.class with name=milk. no need in a separate class